packages/patternlab/styleguide/source/_data/headerSearch.json

```diff
@@ -8,6 +8,17 @@
       "text": "Search",
       "usage": "secondary"
     },
-    "suggestedScopes": ["Department of Conservation & Recreation", "Executive Office of Energy and Environmental Affairs"]
+    "suggestedScopes": [
+      {
+        "value": "Department of Conservation & Recreation",

```

Comment:
```suggestion
        "value": "department-of-conservation--recreation",
```

Reply:
Please update header-search.md to document the new data structure. All values should be "slugified" see the slugify function  https://github.com/massgov/mayflower/pull/1973/files#r2063473220